server/py/services/api/crud/model_monitoring/model_endpoints.py

```diff
@@ -224,6 +205,8 @@ async def create_model_endpoints(
             ]
```

Comment:
fix the `model_endpoints_instructions` type as well. Instead of:
```
tuple[
 mlrun.common.schemas.ModelEndpoint,
 mm_constants.ModelEndpointCreationStrategy,
 str,
]
```

it should be
```
tuple[
   mlrun.common.schemas.ModelEndpoint,
   mm_constants.ModelEndpointCreationStrategy,
]
```